website/docs/cloud-docs/api-docs/project-team-access.mdx

```diff
@@ -4,6 +4,38 @@ description: >-
   Use the `/team-projects` endpoint to manage team access to a project. List, show, add, update, and remove team access from a project using the HTTP API.
 ---
 
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
```

Comment:
Added to fix the broken status code links and any status code links added later.